synapse/lib/gis.py

```diff
@@ -0,0 +1,107 @@
+import math
+
+import synapse.lib.syntax as s_syntax
+
+'''
+Synapse module with helpers for geo-spacial calculations.
+'''
+
+# base geo distances will be in mm
```

Comment:
probably want to note that these constants are for ``#loc.milkway.gouldbelt.localbubble.localfluff.orionarm.sol.earth``.

Reply:
haha, adding "earth based geo-spacial calculations" to the module doc string :D